Proiect_PAO/src/aplicatieBancara/Client/Client.java

```diff
@@ -0,0 +1,119 @@
+package aplicatieBancara.Client;
+
+import aplicatieBancara.Card.Card;
+import aplicatieBancara.Cont.Cont;
+
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+public class Client {
+
+    private static int id = 0;
+    private final int idClient;
+    private String nume;
+    private String prenume;
+    private String CNP;
+    private Date dataNastere;
+    private String email;
+    private String telefon;
+    private Adresa adresa;
+    private ArrayList<Cont> conturi = new ArrayList<Cont>();
+
+    public Client(String nume, String prenume, String CNP, Date dataNastere, String email, String telefon, Adresa adresa) {
+        id  = id + 1;
+        this.idClient = id;
+        this.nume = nume;
+        this.prenume = prenume;
+        this.CNP = CNP;
+        this.dataNastere = dataNastere;
+        this.email = email;
+        this.telefon = telefon;
+        this.adresa = adresa;
+    }
+
+    @Override
+    public String toString() {
+        SimpleDateFormat formatter = new SimpleDateFormat("dd-M-yyyy");
+        return "ID client: " + idClient + "\n" +
+                "Nume: " + nume + "\n" +
+                "Prenume: " + prenume + "\n" +
+                "CNP: " + CNP + "\n" +
+                "Data nasterii: " + formatter.format(dataNastere) + "\n" +
+                "Email: " + email + "\n" +
+                "Telefon='" + telefon + "\n" +
+                "Adresa:" + adresa + "\n";
+    }
+
+    public void adaugaCont(Cont c){
+        conturi.add(c);
+    }
+
+    public void eliminaCont(Cont c){
+        conturi.remove(c);
+    }
```

Comment:
eu as adauga aceste doua operatii in serviciu pentru ca ele tin de logica de business (cu ce se ocupa un service in general) si as lasa definitia obiectelor (clasele pentru obiectele tale din aplicatie) in clasele lor grupate intr-un package general entity/entitati si in acel package poti tine toate tipurile de obiecte. Apoi un alt package servicii (chiar daca ai doar un serviciu)

Reply:
Am scos metodele eliminaCont si adaugaCont si campul conturi din clasa Client. Acum conturile si metodele aferente tin de serviciu. 
Am grupat si serviciile si entitatile in pachete diferite.